### PR TITLE
fixes typo

### PR DESCRIPTION
**Pull request by jonschlinkert**
Changes `GItHub` to `GitHub`

### CODE REVIEW

**Comment by b264** [comment]
`SHip It`

**Comment by lee-dohm** [comment]
Thanks for the contribution. This is a duplicate of #26. See the comments there for the full explanation :+1:

**Comment by jonschlinkert** [comment]
lol sorry about that! for some reason, I assumed no one besides me would create a pr for something so small lol.

**Comment by lee-dohm** [comment]
It's quite alright :grinning:

**Comment by b264** [comment]
I definitely would have if I'd seen it :trollface: :stuck_out_tongue_winking_eye:

File: Policies/amendment-to-github-terms-of-service-applicable-to-u-s-federal-government-users.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ redirect_from:
 
 {{#tip}}
 
-This Amendment to GitHub's [Terms of Service](/articles/github-terms-of-service) applies only to users that are using GitHub on behalf of a United States federal government agency. If you are not using GItHub on behalf of a U.S. federal government agency, the standard [GitHub Terms of Service](/articles/github-terms-of-service) applies to you.
```
Comment (Golfry66) [inline Policies/amendment-to-github-terms-of-service-applicable-to-u-s-federal-government-users.md]: Null

**Pull request closed without merging**